exo-go/src/app_config_helpers/index.go

```diff
@@ -26,20 +27,82 @@ func GetAppConfig() types.AppConfig {
 	return appConfig
 }
 
+// GetEnvironmentVariables returns the environment variables of
+// all dependencies listed in appConfig
+func GetEnvironmentVariables(appConfig types.AppConfig) map[string]string {
+	envVars := map[string]string{}
+	for _, dependency := range appConfig.Dependencies {
+		for variable, value := range dependency.GetEnvVariables() {
+			envVars[variable] = value
+		}
+	}
+	return envVars
+}
+
+// GetDependencyNames returns the names of all dependencies listed in appConfig
+func GetDependencyNames(appConfig types.AppConfig) []string {
+	dependencyNames := []string{}
```

Comment:
Here is a trick that I like doing: call the variable that contains the result of the function `result`. There are several advantages to this:
- the method name is called `GetDependencyNames`, so it is pretty clear that it returns dependency names. No point in saying that again via the variable name here. 
- if you name this variable `result`, it is abundantly clear that it contains what the function returns. With the current name all it says that it contains some sort of dependency names, but its unclear whether these are the ones that ultimately get returned or other dependency names.
- less thinking about what the proper variable name is
- `result` is shorter